image/templates/helm/stackrox-central/templates/01-central-00-db-serviceaccount.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
   annotations:
     {{- include "srox.annotations" (list . "serviceaccount" "central-db") | nindent 4 }}
 imagePullSecrets:
-  {{- range $secretName := ._rox.imagePullSecrets._names }}
```

Comment:
Just for consistency - all other definitions of `imagePullSecrets:` in service accounts had no indentation.
But I'm fine with indenting, as long as we do this consistently! I just leaned towards making the diff smaller and assumed no indentation is preferred since it was more common.